CDP4WebServices.API/Services/BusinessLogic/Implementation/EngineeringModelZipExportService.cs

```diff
@@ -1045,5 +1051,28 @@ private void WriteHeaderToZipFile(ExchangeFileHeader echExchangeFileHeader, ZipF
             credentials.IsParticipant = currentParticipantFlag;
             this.PermissionService.Credentials = credentials;
         }
+
+        /// <summary>
+        /// An implementation of <see cref="IEqualityComparer{Thing}"/> for <see cref="CDP4Common.DTO.Thing"/> to be used in <see cref="HashSet{Thing}"/>.
+        /// </summary>
+        private class ThingComparer : IEqualityComparer<Thing>
```

Comment:
Please have a good look whether this is already available somewhere? And if not for sure move it to own file and maybe call it `ThingIidComparer`.

Reply:
I agree with @alexatrhea that the name should be `ThingIidComparer`. I also think that this belongs in CDP4Common in the SDK as a public class.

A potential place (namespace) to put it in could be `CDP4Common.Comparers`